gui/tools/Helper.h

```diff
@@ -0,0 +1,227 @@
+#include <algorithm>
+#include <vector>
+#include <sstream>
+#include <string>
+
+// VTK includes
+#include <vtksys/SystemTools.hxx>
+#include "vtkMatrix3x3.h"
+#include "vtkMatrix4x4.h"
+#include "vtkNew.h"
+#include "vtkTransform.h"
+
+namespace help
+{
+  //----------------------------------------------------------------------------
+  // Description
+  // Split string by separated char
+  static void SplitString(const std::string &s, char delim,
+                          std::vector<std::string> &elems)
+  {
+    std::stringstream ss(s);
+    std::string item;
+    while (std::getline(ss, item, delim))
+      {
+      elems.push_back(item);
+      }
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Get the filename path (copy from vtk in order to allow debug)
+  static std::string GetFilenamePath(const std::string& filename)
+  {
+    std::string fn = filename;
+    vtksys::SystemTools::ConvertToUnixSlashes(fn);
+
+    std::string::size_type slash_pos = fn.rfind("/");
+    if (slash_pos != std::string::npos)
+    {
+      std::string  ret = fn.substr(0, slash_pos);
+      if (ret.size() == 2 && ret[1] == ':')
+      {
+        return ret + '/';
+      }
+      if (ret.empty())
+      {
+        return "/";
+      }
+      return ret;
+    }
+    else
+    {
+      return "";
+    }
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Read global path and extract all contained path
+  static std::vector<std::string> ExtractAllFilePath(const char* globalPath)
```

Comment:
I got a build warning that this function is unused.  That's good!  Let's remove it.